mbed_lstools/lstools_base.py

```diff
@@ -176,6 +176,7 @@ def __init__(self):
         "9012": "SEEED_TINY_BLE",
         "9900": "NRF51_MICROBIT",
         "FFFF": "K20 BOOTLOADER",
+        "OSHW":"Unknown DAPLink Implementation",
```

Comment:
`UNKNOWN_DAPLINK` would work. Its not an unknown target, its a known identifier to a group of Daplink boards that do not have mbed platform support.